packages/openapi-2-kong/package.json

```
@@ -29,5 +29,8 @@
     "webpack": "^4.42.1",
     "webpack-cli": "^3.3.11"
   },
+  "repository": "https://github.com/Kong/insomnia/tree/develop/packages/openapi-2-kong",
+  "bugs": "https://github.com/Kong/insomnia/issues",
```

According to [the docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bugs), this needs to be the following:

```suggestion
  "bugs": {
    "url": "https://github.com/Kong/insomnia/issues"
  },
```

---

Thanks for looking up! I based my changes on an old convention, obviously outdated. Fixed now.